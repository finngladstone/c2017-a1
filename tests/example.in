place
place 7
place 7 
place 71  
place x y
place xy
resign
