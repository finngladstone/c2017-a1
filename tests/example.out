Invalid!
Invalid coordinate
Invalid!
Invalid!
Invalid!
Invalid coordinate
White wins!

Thank you for playing!
